docs/design/autoware-interfaces/ad-api/list/api/planning/factors.md

```diff
@@ -0,0 +1,19 @@
+# /api/planning/factors
```

Comment:
I think it's better to make it more specific.

```suggestion
# /api/planning/behavior_factors
```

Reply:
In the directory structure, `avoidance` and `surround obstacle` are located under `motion_planning`. So I think it's not only behavior.

Reply:
I meant the vehicle's general behavior. TIER IV's behavior/motion planning is irrelevant.

Reply:
Split into `velocity_factors` and `steering_factors`.